### ISSUE TITLE
[Suggestion] Add a FAQ to the CoC

### ISSUE THREAD

**Issue by CristinaMendonca** [ContributorCovenant/contributor_covenant#575]
I've seen (in Reddit) that some people are misunderstanding the CoC and/or anticipating CoC abuse. I think a good approach to deal with these reactions would be to add a diplomatic, pedagogic FAQ to the CoC addressing the most common questions and potential issues (e.g., what happens if you didn't know the word you were using was offensive to someone somewhere, how you should react if you were unjustly accused of violating the CoC). This way, it would be easier to fend off criticism by simply asking people to read the CoC carefully, including the FAQ.

**Comment by CoralineAda** [comment]
I love this idea. Is this something that you would be interested in working on?

**Comment by CristinaMendonca** [comment]
Sure, I can at least give it a try! :)

**Comment by CristinaMendonca** [comment]
Ok, I don't have time to spend much more on this, so I will just leave here a suggestion and hopefully someone can heavily improve on it and actually implement it. I tried to make the text pedagogic and diplomatic, but some people may consider it too soft. My personal opinion is that we stand to gain the most by trying to make the CoC focus on specific behaviors to abolish and to promote, as well as progressive increase of knowledge about what is or is not a hostile or a welcoming environment, so as to avoid as much knee-jerk reactions as possible. But you're free to do with this idea as you may :)

**What if someone unjustly accuses me of violating the CoC?**
The first step when dealing with accusations of violation of the CoC is to deal with it in a professional manner. Abusers of the CoC will probably attempt to use your frustration or anger in their advantage. So do not use sarcasm, ridicularization, attack the CoC or its enforcement, or the accuser, - calmly focus on the evidence of what happened and explain your intentions clearly (e.g., if accused of offending someone, make it clear that that was not your intention). The more you stick to the facts of what happened, show yourself as willing to help the administrators evaluate the claims, and the less fuel you give to your unjust accuser, the better. Do note that this does not mean you shouldn't criticize the CoC, but obviously the best time to do it is not when you're accused of violating it.

**Won't this damage the end product?**
As far as we know, there is no solid evidence that can be put forward to show that the CoC damages or does not damage the end product. If people who work better in hostile environments are also those that produce better products, then the CoC will have a negative impact, - but there's no reason to expect such causal relationship between hostile environments and product quality. It is more probable the CoC will not affect end product, but will affect the experience of those that participate in communities where the CoC is enforced.

**Won't this just promote participation for the sake of participation and an end of meritocracy?**
The CoC is not a positive discrimination policy, it does not include any recommendation on how to recruit, select or otherwise manage human resources. By fostering a more cooperative and civil environment, the CoC actually promotes that everyone reveals and develops their capacities in contributing, regardless of their specific capacity to fit in in hostile environments.

**Why should me/my community have to change to accommodate this one person?**
The level of accommodation that the CoC implies is not higher than the one that is expected when dealing face-to-face with others in professional settings. It is not about accommodating to one person, it's about making all collaborators feel welcome or at the very least not making them feel unwelcome.

**When I code, I just see the code, not the person, so why should I care about this CoC? Can't minorities keep their minority-status to themselves and focus on the code?**
The CoC's focus is on people addressing each other in a civil and cooperative manner, not on whether people should or should not reveal their minority status. Yet, the CoC does imply that people who reveal their minority-status should be treated in a civil way, - much like everyone else.

**Isn't this just a way for progressive/left-wing people to push their agenda?**
"Everything is politics", but the CoC is not about an advance of progressive/left-wing politics, it's about establishing a minimal level of civil collaboration. Conservative/right-wing people certainly also care about professionalism and civility in the workplace, be it online or offline.

**This just enforces political correctness, it won't lead to any real change.**
The CoC is framed as "a small step forward". It certainly will not lead to attitude change in any large scale, but that's not its objective. The objective is simply to create more civil and professional environments, for both majority and minority members.

**What is so wrong that white males dominate a certain profession? People never care with female dominated areas, nor do they take into account that maybe minorities are simply not interested in a certain profession.**
The problem is that this dominance implies (which is not proof, of course) that there is a difference in opportunities. Upon further look, it is not difficult to find causes that may be leading to differences in opportunities, such as stereotypes that posit that certain groups are less competent or should not be interested in certain topics, etc. The same, of course, applies to the access of males to female dominated professions (and any general gender equality approach needs to deal with both), but that is outside the scope of the CoC, as it deals with software development.

**Comment by tbird20d** [comment]
I would add:

**Does the prohibition on publishing private information include email addresses sent to a public list?**
No.  Information that has voluntarily been published to a public location does not fall under the category of private information.  Such public information may be used within the context of the project according to project norms (such as in commit meta-data in code repositories), without that constituting a breach of the CoC.

**Comment by ShadowGoat** [comment]
Yeah this is pretty cancer and about sums up what I thought of this

**Comment by tbird20d** [comment]
I'm confused @ShadowGoat. Are you referring to my addition to the FAQ clarifying email handling, or are you naysaying the FAQ, or the whole contributor covenant?  If you have a specific, constructive complaint, then please share it.  If just negative musings, then never mind.
